tests/integration/test_model.py

```diff
@@ -77,66 +79,68 @@ def test_training_and_prediction(c, training_df):
         )
     """
     )
-
     check_trained_model(c)
 
 
 @pytest.mark.gpu
 def test_cuml_training_and_prediction(c, gpu_training_df):
-    model_query = """
+    c.sql(
+        """
         CREATE OR REPLACE MODEL my_model WITH (
-            model_class = 'cuml.linear_model.LogisticRegression',
+            model_class = 'LogisticRegression',
```

Comment:
Now that this test has been generalized, is it possible to remove the `pytest.mark.gpu` and run on both CPU and GPU? This question also applies to the other tests marked for GPU here

Reply:
I ended up not doing this because of the `training_df` and `gpu_training_df` fixtures. There's a series of commits on this PR where I tried a couple of different ways but was having trouble getting it to work, and I didn't necessarily want to modify the fixtures themselves because that would require also modifying other unrelated tests.

I guess another solution is to just not use the fixtures and have a `gpu` param instead to create the tables within the test. Does that sound ok?

Reply:
Would it work if we merged `training_df` and `gpu_training_df` into a single fixture that registers both tables (skipping the GPU case if dask-cudf isn't available) and then used the `gpu` param to select which table is used to create the model?